django_project/lesson/models/specification.py

```diff
@@ -0,0 +1,48 @@
+# coding=utf-8
+"""Specification model definitions for lesson apps.
+
+"""
+import logging
+
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from lesson.models.worksheet import Worksheet
+
+logger = logging.getLogger(__name__)
+
+
+class Specification(models.Model):
+    """Worksheet lesson model."""
```

Comment:
A specification is a particular requirement for a task that the learner is required to do in the lesson.